Automattic/newspack-blocks
# Add audio block

### All Submissions:






### Changes proposed in this Pull Request:



This adds an audio block, which can display either a static audio file:

![image](https://user-images.githubusercontent.com/7383192/75779194-797e1500-5d59-11ea-847a-8d6bfc386608.png)

or a latest podcast episode, using an RSS feed:

![image](https://user-images.githubusercontent.com/7383192/75779706-63bd1f80-5d5a-11ea-9af6-70ce9b783000.png)


### How to test the changes in this Pull Request:

1. Edit a post or a page
1. Observe that there is a new block available - Newspack Audio block (there's also a Core Audio block)
1. Insert the block 
1. Add an podcast RSS feed in block "RSS Feed" tab of settings (e.g. `http://feeds.revealradio.org/revealpodcast`)
1. Observe that the latest episode is fetched and loaded to play
1. Remove RSS feed, add and audio file source in "Settings" tab of block settings
1. Observe that the file is fetched and loaded to play

### Other information:
